src/res/translation/translation_de_DE.ts

```diff
@@ -154,7 +154,7 @@
     <message>
         <location filename="../../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2021 The Jamulus Development Team</source>
-        <translation type="unfinished"></translation>
+        <translation>Copyright (C) 2005-2021 beim Jamulus Entwicklungs Team</translation>
```

Comment:
This started some [discussion](https://github.com/jamulussoftware/jamulus/pull/1097#discussion_r581265793) if this should be translated at all.

Personally, I think I'd keep it untranslated, but lets wait what the discussion yields.